inst/templates/ad_adlb.R

```diff
@@ -228,6 +222,74 @@ adlb <- adlb %>%
   derive_var_pchg()
 
 
+## Calculate lab grading ----
+
+# Assign ATOXDSCL and ATOXDSCH to hold lab grading terms
+# ATOXDSCL and ATOXDSCH hold terms defined by NCI-CTCAEv4.
```

Comment:
I think it's also good to add a comment in the template to note that the default `atoxgr_criteria_ctcv4` metadata stored in admiral was used in the derivation for NCI-CTCAEv4, and users can specify their own metadata if needed.

Reply:
I added some text, let me know if its sufficient.